bridge/slack/handlers.go

```diff
@@ -61,7 +60,16 @@ func (b *Bslack) handleSlackClient(messages chan *config.Message) {
 		case *slack.OutgoingErrorEvent:
 			b.Log.Debugf("%#v", ev.Error())
 		case *slack.ChannelJoinedEvent:
+			// When we join a channel we update the full list of users as
+			// well as the information for the channel that we joined as this
+			// should now tell that we are a member of it.
 			b.populateUsers()
+
+			b.channelsMutex.Lock()
+			defer b.channelsMutex.Unlock()
```

Comment:
This will never get unlocked?
Should be unlock() after you add the channel

Reply:
🤦‍♂️